examples/react-demo/src/app/page.tsx

```diff
@@ -35,7 +34,7 @@ createEthSSOModal({
     // { url: "mpetrunic.eth" },
     // { url: "sso.wallet.connect.com" },
   ],
-  redirectUrl: "http://localhost:3001",
+  redirectUrl: "http://localhost:3001/callback/",
```

Comment:
do you really need /callback?
Couldn't sdk when initialized check if URL contains redirect params?

Reply:
But in oAuth callback is backend endpoint, right? Considering we are on frontend, don't think it makes sense to create frontend routing for that?

Reply:
no its actually on frontend, that route process data and doo what need to do like storing in local storage or in our case sending post message